skynet/modules/ttt/summaries/v1/models.py

```diff
@@ -14,7 +12,7 @@ class HintType(Enum):
 
 class DocumentPayload(BaseModel):
     text: str
-    hint: HintType = summary_default_hint_type
+    hint: HintType = HintType.TEXT
```

Comment:
Ah I see. Good call. Arguably though, we focus more on conversations, so maybe the other default is better?

Reply:
currently there's hardly and difference between them, if at all

Reply:
Ok.